tests/settings.py

```diff
@@ -13,4 +15,5 @@
 OIDC_AUTH = {
     'OIDC_ENDPOINT': 'http://example.com',
     'OIDC_AUDIENCES': ('you',),
+    'CREATE_USER': os.getenv('CREATE_USER', 'yes') == 'yes',
```

Comment:
Wouldn't this make more sense as a boolean?

Reply:
This is a boolean, right?  It evaluates to true or false.  In my mind it's hard to pass in an env var as a boolean.  So in python, it's a boolean.  As an env var it's a string.  If one wants to use the string 'True' or the string 'False', okay - maybe that would be clearer.  Or maybe not using the string 'True', it makes one think, and see what's happening.  I'm using the same approach as found in many Django books, here.  Thanks for your comments here.  It's good to discuss.  Hope that makes sense.

Reply:
Hehe, whoops :sweat_smile: Must have been that Friday afternoon slip :see_no_evil: Never mind the comment